### PR TITLE
feat: add dark theme

### PR DESCRIPTION
**Pull request by PhilippeBarre**
I added the dark theme with a class on the body which is toggle via a checkbox

image

Issue : Add dark theme in app #9

### CODE REVIEW

File: toggle.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 30px;
+  background-color: rgba(0, 0, 0, 0.25);
```
Comment (gouri-panda) [inline toggle.css:6]: @PhilippeBarre it doesn't look good. Can you mix up with the background?

**Review by gouri-panda — CHANGES_REQUESTED** [review]

File: toggle.css (hunk 1 above)
Comment (PhilippeBarre) [reply toggle.css:6]: image

Hmm sorry, what doesn't look good ?

This background-color is the color of the toggle when it is unchecked. So it's black, transparent to see the image behind it.

Comment (PhilippeBarre) [reply toggle.css:6]: I will change the active color to accord with your theme
image

**Review by gouri-panda — APPROVED** [review]
@PhilippeBarre Thank you for the PR🎉.

**Pull request merged**